### PR TITLE
Move GCP upload script to a separate package

### PR DESCRIPTION
**Pull request by mattsoulanille**

### CODE REVIEW

File: custom_nodes/.changeset/chilled-rings-brake.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
```
Comment (jinjingforever) [inline custom_nodes/.changeset/chilled-rings-brake.md:1]: What is this file?

**Review by jinjingforever — APPROVED** [review]
Thank you!

File: custom_nodes/.changeset/chilled-rings-brake.md (hunk 1 above)
Comment (mattsoulanille) [reply custom_nodes/.changeset/chilled-rings-brake.md:1]: This file is a changeset that records what this PR changed, what packages are affected, and how those packages' versions should be bumped (major / minor / patch). When we publish, I run `changeset version`, and it automatically updates all the package.json files with the correct version increase. Then, it deletes all these pending changeset files and puts them in the changelog for each package.

Comment (jinjingforever) [reply custom_nodes/.changeset/chilled-rings-brake.md:1]: nice!!

**Pull request merged**